data/Tusharjagi.json

```diff
@@ -1,7 +1,29 @@
 {
   "name": "Tushar Jagi",
   "type": "personal",
-  "bio": "I'm Front End Developer starting journey in open source",
+  "bio": "JavaScript developer | **MERN Stack** | openSource | Enthusiast in BlockChain",
+  "tags": [
+    "openSource",
```

Comment:
It'd be nice to Have it as "Open Source" instead of "openSource"

Reply:
Nothing wrong with `openSource`, but the tag `Open Source` is more widely used among the LinkFree users. So it may improve the searchability of your profile.